.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,65 @@
+name: "🐞 Bug Report"
+description: File a bug report
+title: "[BUG]: <Title>"
+labels: ["kind/bug", "status/triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+        If you suspect this could be a bug, follow the template.
+  - type: textarea
+    attributes:
+      label: What version of Dgraph are you using?
+      description: Copy and paste all you see using the command 'dgraph version'
+      placeholder: |
+        Paste here.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Tell us a little more about your go-environment?
+      description: Copy and paste all you see using the command 'go version'
+      placeholder: |
+        Paste here.
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Have you tried reproducing the issue with the latest release?
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: What is the hardware spec (RAM, CPU, OS)
+      description: Share all possible context of your side.
+      placeholder: |
+        Paste here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: What steps will reproduce the bug?
+      description: Enter details about your bug (command/config used to run Dgraph).
+      placeholder: |
+        1. Using the command curl https://get.dgraph.io -sSf | VERSION=v20.03.1-beta1 bash
+        2. Started Dgraph locally(or docker for example) With this config...
+        3. Using the client X
+        4. See error...
```

Comment:
I think you did not understand. This is not something related to get.dgraph. It's just an example. This is a placeholder tip.

All steps, no matter which steps are taken, are important to get context and thus determine if it is a valid bug or not. Perhaps I should emphasize that this is an example if it wasn't clear. But when it comes to rederrizing yaml, it may be understandable that this is an example of steps to be explained.